ray-operator/config/rbac/role.yaml

```diff
@@ -54,7 +54,7 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - serviceaccount
+  - serviceaccounts
```

Comment:
why do we need this change? `serviceaccount` should be correct.

Reply:
If we need `serviceaccount`, we should update code
https://github.com/ray-project/kuberay/blob/7ecdd22c5812db8754e470635b20d9ee4a6a425d/ray-operator/controllers/ray/raycluster_controller.go#L77
https://github.com/ray-project/kuberay/blob/7ecdd22c5812db8754e470635b20d9ee4a6a425d/ray-operator/controllers/ray/rayservice_controller.go#L48
`// +kubebuilder:rbac:groups=core,resources=serviceaccount,verbs=get;list;watch;create;delete`

Reply:
It turns out the plural form is the correct way to specify the resource name (compare to "pods" elsewhere in this file).
With "serviceaccount" (singular) it fails to grant the necessary permissions (which are used only to set up the autoscaler's service account)

Reply:
> If we need `serviceaccount`, we should update code

Thanks for pointing that out!! Will do.